difftest/t1-simulator/readme.md

```diff
@@ -0,0 +1,11 @@
+## Build
```

Comment:
Please add more detail readme on the difftest, including the `rational.md`.
- the architecture of the difftest design.
- the public interface and usage of the difftest, aka the schema of all events.
Please do it in the following PR.